tests-performance/src/template-with-cli-args.cc

```diff
@@ -0,0 +1,67 @@
+#include <mata/inter-aut.hh>
+#include <mata/nfa.hh>
+#include <mata/mintermization.hh>
+#include <iostream>
+#include <fstream>
+#include <chrono>
+#include <string>
+#include <cstring>
+
+using namespace Mata::Nfa;
+
+int main(int argc, char *argv[])
+{
+	if (argc != 2) {
+		std::cerr << "Input file missing\n";
+		return EXIT_FAILURE;
+	}
+
+	std::string filename = argv[1];
+
+	std::fstream fs(filename, std::ios::in);
+	if (!fs) {
+		std::cerr << "Could not open file \'" << filename << "'\n";
+		return EXIT_FAILURE;
+	}
+
+	Mata::Parser::Parsed parsed;
+	Nfa aut;
+    Mata::StringToSymbolMap stsm;
+	const char* nfa_str = "NFA";
```

Comment:
I would create a `constexpr std::string`.

Reply:
Sadly, in C++-17, it seems we cannot use `constexpr std::string` only `std::string`, so I will leave it as just `std::string`. Apparently `constexpr` strings are supported from C++20 only.